ghenet-Ghenet/monument-app/src/Blog/Header/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Nav.css';
+
+const nav = ()=>{
+    return (
+        <div>
+        <header className="vertical-center1">
+            <div>
+                <h1><a href="http://localhost:3000/">Monument1</a></h1>
```

Comment:
This is ab absolute path to local host. You should never do this. Use a relative path to ``` htef="/[whatever the url is]"``` . This way it will wotk off your machine.